commons-mfa/src/main/java/org/exoplatform/mfa/rest/mfa/MfaRestService.java

```diff
@@ -44,6 +51,32 @@ public Response getMfaSystem() {
       return Response.serverError().build();
 
     }
+  }
+
+  @Path("/changeMfaFeatureActivation/{status}")
+  @PUT
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @ApiOperation(value = "Switch the Activated MFA System", httpMethod = "PUT", response = Response.class, produces = MediaType.APPLICATION_JSON)
+  @ApiResponses(value = { @ApiResponse(code = HTTPStatus.OK, message = "Request fulfilled"),
+      @ApiResponse(code = HTTPStatus.UNAUTHORIZED, message = "Unauthorized operation"),
+      @ApiResponse(code = HTTPStatus.INTERNAL_ERROR, message = "Internal server error"), })
+  public Response changeMfaFeatureActivation(@ApiParam(value = "Switch the Activated MFA System to avtivated or deactivated", required = true) String status) {
+    boolean isActiveBool = Boolean.parseBoolean(status);
+    featureService.saveActiveFeature(MFA_FEATURE, isActiveBool);
```

Comment:
donc call featureService directly.
We should have not intelligence in RestService.
do something like 
mfaService.saveActiveFeature which will call FeatureService